### PR TITLE
Typesafe config readme

### PR DESCRIPTION
**Pull request by jamesgorrie**

### CODE REVIEW

File: typesafe_config_base/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Base container image for typesafe services at Wellcome.
+
+## Usage
+
+This container is intended to be extended from like so:
+
+```
+FROM wellcome/typesafe_config_base:104
+
+ADD target/universal/stage /opt/docker
```
Comment (alexwlchan) [inline typesafe_config_base/README.md]: Possibly worth an explanation and/or example of what this `target` folder is?

File: typesafe_config_base/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# wellcome/typesafe_config_base
+
+Base container image for typesafe services at Wellcome.
```
Comment (alexwlchan) [inline typesafe_config_base/README.md]: What is a “typesafe service”?

How about “for Scala apps that use Typesafe (link) for config”?

File: typesafe_config_base/README.md (hunk 1 above)
Comment (jamesgorrie) [reply typesafe_config_base/README.md]: Do you think? Isn't that tied into SBT / Typesafe config?
Which the link describes?

Comment (alexwlchan) [reply typesafe_config_base/README.md]: I think now you’ve clarified that it’s a Scala app, this is fine.

(I wrote this comment first.)

**Pull request merged**